backend/danswer/danswerbot/slack/listener.py

```diff
@@ -471,8 +471,7 @@ def _initialize_socket_client(socket_client: SocketModeClient) -> None:
                         embedding_model = get_current_db_embedding_model(db_session)
                         if embedding_model.cloud_provider_id is None:
                             warm_up_encoders(
```

Comment:
just pass in the embedding model object instead of pulling out all the info and passing it 1 by 1